src/components/api.js

```diff
@@ -0,0 +1,223 @@
+export default class Api {
+  constructor(options) {}
+
+  //methods
+  getInitialCards() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+      headers: {
+        authorization: "4792ec92-cf1c-45a6-8740-0f5d63585faa",
+      },
+    })
+      .then((res) => {
+        if (res.ok) {
+          //check for successful response
+          console.log(res.ok);
+          return res.json();
+        }
+        //check for unsuccessful response
+        console.log("Error");
+        return Promise.reject(`getInitialCards method Error: ${res.status}`);
+      }) //process errors
+      .catch((err) => console.error(err));
```

Comment:
It’s better not to use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request, and this end is in `index.js`  after all blocks `then`. And in real projects blocks `catch` usually open error popups and so on.